stackdriver_trace.c

```diff
@@ -599,7 +604,8 @@ PHP_FUNCTION(stackdriver_trace_list)
     int i = 0;
     stackdriver_trace_span_t *trace_span;
     int num_spans = STACKDRIVER_TRACE_G(spans)->nNumUsed;
-    zval labels[num_spans], spans[num_spans];
+    zval *labels = emalloc(num_spans * sizeof(zval));
```

Comment:
Good point, I'm calling `array_init` twice. Each one allocs more memory, so we're leaking memory here. I'll address this in a separate PR.